lib/tasks/retire.rake

```diff
@@ -64,6 +64,29 @@ namespace :retire do
     )
   end
 
+  desc "Publish transaction start page via publishing api"
```

Comment:
This needs to be moved to `lib/tasks/publishing_api.rake` retire namespace is reserved for rake task that are part or concern retiring smart answers. 

Also we require tests to be add to rake tasks